_locales/en/messages.json

```diff
@@ -481,6 +494,31 @@
     "message": "Remove section",
     "description": "Label for the button to remove a section"
   },
+  "setLinterLink": {
+    "message": "See a full list of rules",
+    "description": "Stylelint or CSSLint rules label added before a link"
+  },
+  "setLinterRulesTitle": {
+    "message": "Set rules for $linter$",
+    "description": "Stylelint or CSSLint popup header",
+    "placeholders": {
+      "linter": {
+        "content": "$1"
+      }
+    }
+  },
+  "resetLinterRules": {
+    "message": "Reset",
+    "description": "Reset Stylelint or CSSLint rules"
```

Comment:
And the description should be copied from other generic labels e.g. "Used in various parts of UI to indicate that something may be reset to its original state"